doc/rst/source/explain_-J_full.rst_

```diff
@@ -273,15 +273,20 @@ projection names stands for Equal-Area and Conformal, respectively):
 
 .. _-Jp_full:
 
-    **-Jp**\ *scale*\ [**+a**]\ [**+o**\ *origin*][**+r**\|\ **z**] or **-JP**\ *width*\ [**+a**]\ [**+o**\ *origin*][**+r**\|\ **z**]
+    **-Jp**\ *scale*\ [**+a**]\ [**+f**\ [**e**\|\ **p**\|\ *radius*]][**+r**\ *offset*][**+t**\ *rotation*][**+z**\ [**p**\|\ *radius*]]] or **-JP**\ *width*\ [**+a**]\ [**+f**\ [**e**\|\ **p**\|\ *radius*]][**+r**\ *offset*][**+t**\ *rotation*][**+z**\ [**p**\|\ *radius*]]
     (Polar coordinates (*theta*, *r*))
 
-    Optionally add **+a** if *theta* is azimuths CW
-    from North instead of directions CCW from East [Default]. Optionally
-    append **+o**\ *origin* in degrees to indicate a CW (or CCW if **+a** iset set) angular offset [0].
-    Finally, append **+r** if *r* represents elevations in degrees (requires *south* >= 0
-    and *north* <= 90), or append **+z** if you want to annotate depth rather than
-    radius [Default]. Give *scale* in UNIT/r-unit.
+    Give *scale* in UNIT/r-unit.  Optionally add **+a** if *theta* is azimuth CW
+    from North instead of direction CCW from East [Default].
+    Select **+f** to flip the radial direction to point inwards, and append **e** to indicate
+    that *r* represents elevations in degrees (requires *south* >= 0 and *north* <= 90),
+    **p** to select current planetary radius as maximum radius [*north*], or *radius*
+    to specify such a radius directly.
+    Append **+r**\ *offset* to include a radial offset [0].
+    Append **+t**\ *rotation* in degrees to indicate a CCW angular offset [0].
```

Comment:
> Append **+t**\ *rotation* in degrees to indicate a CCW angular offset [0].

It can be CW or CCW depending on the **+a** modifier. Maybe say:

> Append **+t***theta* in degrees to indicate the angle of the East direction or North direction if **+a** is used [0].